vm/qemu/qemu.go

```diff
@@ -142,14 +143,17 @@ func (inst *instance) Boot() error {
 		"-smp", "sockets=2,cores=2,threads=1",
 		"-usb", "-usbdevice", "mouse", "-usbdevice", "tablet",
 		"-soundhw", "all",
+		"-initrd", inst.cfg.Initrd,
 	}
 	if inst.cfg.Kernel != "" {
 		args = append(args,
 			"-kernel", inst.cfg.Kernel,
-			"-append", "console=ttyS0 root=/dev/sda debug earlyprintk=serial slub_debug=UZ "+inst.cfg.Cmdline,
+			"-append", "\"console=ttyS0 root=/dev/sda debug earlyprintk=serial slub_debug=UZ\" "+inst.cfg.Cmdline,
 		)
 	}
 	qemu := exec.Command(inst.cfg.Bin, args...)
+
+	log.Printf("qemu command : ", qemu.Args)
```

Comment:
Surround it with:
```
  if inst.cfg.Debug {
    ...
  }
```

Then it will be printed if syz-manager is started with -debug flag, which is meant to debug e.g. qemu startup issues.
It's too much irrelevant output for normal operation (esp if I have several dozens of qemu instances).